build_package.py

```diff
@@ -48,6 +50,10 @@ def travis_build_package():
         print("Version must be a valid PEP440 version (version is: {})".format(version))
         return "Version must be a valid PEP440 version (version is: {})".format(version)
 
+    if name.lower() in omitted_release_packages:
+        print("The input package {} has been disabled for release from Travis.CI.".format(name))
+        return "The input package {} has been disabled for release from Travis.CI.".format(name)
```

Comment:
Empty return

If return has a message, it will fail the Travis build. It's probably not an error to not release a package on purpose